README.md

```diff
@@ -63,6 +63,42 @@ RKE2_URL=https://myserver:6443 RKE2_TOKEN=XXX ./install.sh
 
 We provide a simple automated way to install RKE2 on AWS via terraform scripts, this method requires terraform to be installed and access to AWS cloud, to get started please checkout the [rke2-build](https://github.com/rancher/rke2-build) repo.
 
+## CentOS/RHEL 7 RPM Installation
+
+Signed RPM's are published for RKE2 within the `rpm.rancher.io` RPM repository. In order to use the RPM repository, on a CentOS 7 or RHEL 7 system, run the following bash snippet:
+
+```bash
+cat << EOF > /etc/yum.repos.d/rancher-rke2-1-18-latest.repo
```

Comment:
We should consider offering a rke2-release package like EPEL's [epel-release](https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F) file that contains both the .repo files and GPG keys. This would make repo installation a one-liner: `rpm -i http://get.k3s.io/rke2-release.rpm`

Reply:
I agree that this is something we should eventually provide, but for now, with the RPM repository being in such a state of flux, I am following steps similar to what the Kubernetes project does as evidenced here: https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-using-native-package-management